cmsimple/tplfuncs.php

```diff
@@ -249,14 +249,28 @@ function searchbox()
 }
 
 
+/**
+ * Returns a link to the homepage.
+ *
+ * @param string $img Optional image file name, searched in template folder.
+ *
+ * @return string HTML
+ */
+function homelink($img = '') 
```

Comment:
For the sake of compatibility with CMSimple templates, the name `homelink` is probably okay, but otherwise the same naming convention as noted above for `img_or_text` should apply.